apinotes/ModelIO.apinotes

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
They will ship apinotes eventually in the Framework, meanwhile including them allows us to merge this change here, and include it in the previews.